etlman/templates/base.html

```diff
@@ -33,11 +33,15 @@
   <!-- place project specific Javascript in this file -->
   <script defer src="{% static 'js/project.min.js' %}"></script>
 
-
+  <script src="{% static 'js/htmx.min.js' %}" defer></script>
+  <script src="{% static 'js/debug.js' %}" defer></script>
```

Comment:
Maybe conditionally include this only during development? https://stackoverflow.com/questions/25783459/how-to-check-debug-true-false-in-django-template-exactly-in-layout-html

Reply:
Done 👍🏽